apps/server/src/modules/tldraw/tldraw.config.ts

```diff
@@ -0,0 +1,22 @@
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { NodeEnvType } from '@src/modules/server';
+
+export interface TlDrawConfig {
```

Comment:
Just a cosmetics, but please keep the same naming convention across the module, use either _TlDraw_ or _Tldraw_ everywhere.